dotcom-rendering/cypress/integration/parallel-3/article.interactivity.spec.js

```diff
@@ -52,9 +52,8 @@ describe('Interactivity', function () {
 			it('should change the list of most viewed items when a tab is clicked', function () {
 				cy.visit(`/Article?url=${articleUrl}`);
 				cy.contains('Lifestyle');
-				cy.get('[data-component="most-popular"]').scrollIntoView({
-					offset: { top: 150 },
-				});
+				// Scroll to bottom to trigger hydration
+				cy.scrollTo('bottom', { duration: 300 });
```

Comment:
Was this change required due to tests failing, or was the goal just to generally improve the consistency of this test?

Reply:
I added these changes because the tests failed without them now that we're actually deferring hydration